test/permissions/remote.provider.create.js

```diff
@@ -0,0 +1,204 @@
+        Category: categories.Document,
+      };
+
+      await login(users.S2.InsertOnlyUser);
+      isError(test, remoteProvider, 'Catalog', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission for category with subsystem, sufficient permission',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'person3S1',
+        Subsystem: subsystems.S1,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      await isSuccess(test, remoteProvider, 'Person', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission for category with catalog, sufficient permission',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'document3C1',
+        Catalog: catalogs.C1,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      await isSuccess(test, remoteProvider, 'Document', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission for category with catalog and subsystem, sufficient permission',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'personDocumentC2S1',
+        Catalog: catalogs.C2,
+        Subsystem: subsystems.S1,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      await isSuccess(test, remoteProvider, 'PersonDocument', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission, no insert permission for category with subsystem/catalog',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'person2S2',
+        Subsystem: subsystems.S2,
+      };
+
+      await login(users.S2.InsertOnlyUser);
+      isError(test, remoteProvider, 'Person', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission, no insert permission for subsystem',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'person2S2',
+        Subsystem: subsystems.S2,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      isError(test, remoteProvider, 'Person', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission, no insert permission for catalog',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'document2C2',
+        Catalog: catalogs.C2,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      isError(test, remoteProvider, 'Document', record);
+    }
+  );
+
+  test.test(
+    'remoteProvider.create: check insert permission, no insert permission for catalog and subsystem',
+    async (test, { remoteProvider, login }) => {
+      const record = {
+        Name: 'personDocument2C1S2',
+        Catalog: catalogs.C1,
+        Subsystem: subsystems.S2,
+      };
+
+      await login(users.S1.InsertOnlyUser);
+      isError(test, remoteProvider, 'PersonDocument', record);
+    }
+  );
+};
+
+module.exports = test;
```

Comment:
Here and in all of the other files in this directory, you can avoid declaring the `test` variable and assign the function to `module.exports` since there is no code that modifies the variable after the declaration.